docs/data-processing.md

```diff
@@ -76,7 +76,7 @@ There is a separate mapping file for `projects` data and `credits` data.
 Some properties either aren't included in the raw data or inferring their value requires special processing.
 In these cases, a `null` value is recorded in the column mapping files and the property is populated using registry-specific logic.
 
-## Protocol Mapping & Categorization
+## Protocol Mapping
```

Comment:
nit: should we pick one of either `Sentence case` or `Title Case` and use consistently for all headings? right now we're using a combo

Reply:
just updated all headers to use title casing.